configure.ac

```diff
@@ -4554,7 +4554,7 @@ fi
     echo "MONO_CORLIB_VERSION = $MONO_CORLIB_VERSION" >> $mcs_topdir/build/config.make
```

Comment:
Why is this necessary ? People have been using PROFILE for years, ie. make all PROFILE=net_4_x. There might be build scripts etc lying around doing the same.